pkg/secrets_reflector/fakes.go

```diff
@@ -0,0 +1,40 @@
+package secretsreflector
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/dcos/client-go/dcos"
+)
+
+// fakeDCOSSecretsClient implements DCOSSecretsClient and is used for
```

Comment:
```suggestion
// fakeDCOSSecretsClient implements DCOSSecretsClient and is used for testing.
```

Reply:
As long as it's within a reasonable line length (80? 120?) I agree.

Reply:
The rule has been to only break after a full sentence (i.e. after `.`). But I am not really adamant here.